packages/account/src/Constants/routes-config.js

```diff
@@ -22,7 +22,7 @@ import {
 const Page404 = React.lazy(() => import(/* webpackChunkName: "404" */ 'Modules/Page404'));
 
 // Order matters
-const initRoutesConfig = ({ is_appstore }) => [
+const initRoutesConfig = ({ is_appstore }, is_social_signup) => [
     {
```

Comment:
since the is_social_signup takes time to has value, the `email and passwords` change to `password' after a bit.